src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -728,6 +730,51 @@ public async Task<bool> WriteConst(LiteralExpressionSyntax constValue, MonoSDBHe
             return false;
         }
 
+        public bool WriteNumber(object number, bool convertToNegative=false)
```

Comment:
Why is `convertToNegative` needed here, instead of changing the `number` as needed when calling this method?

Reply:
nvm!